### ISSUE TITLE
Accessibility Sweep

### ISSUE THREAD

**Issue by rosejbon** [neontribe/contemplating-action#153]
## Description
What needs doing?

## Plan
- [ ] Discuss how how to do this
- [x] @Nikomus to do a quick sweep of the CA app to see what you think
- [ ] Talk about doing an internal neontribe accessibility checklist

## Checklist
- [ ] Create checklist

## Issues
- We could use a skip link that appears on focus to skip over the navigation, to the main content.

- Entire nav isn't in a nav element. Not everything has to be, but it seems odd to have an adjacent nav element (the survey) not be included. This will have originally been because in desktop view for the haven, this content is a non-linked phone number.

- We should use `<main>` in preference to `<div role="main">`, possibly `<main role="main">` for old browsers

- A global footer likely shouldn't be nested inside the role="main" element

- Links should be underlined, rather than differentiated by colour (black and white filter on the footer shows no distinction)
![image](https://user-images.githubusercontent.com/32434620/61624716-111e6f80-ac71-11e9-9f9c-8ca451e4d7b7.png)

- Story cards have the same link address four times, with a variety of descriptions. It may be better to  link the whole card, aria hide its contents, and provide a single SR-accessible description of the link.
![image](https://user-images.githubusercontent.com/32434620/61624728-1b406e00-ac71-11e9-993e-954814f21b29.png)

- heading link underline is partially clipped off
![image](https://user-images.githubusercontent.com/32434620/61624742-21cee580-ac71-11e9-9137-38773d0db651.png)

- In some cases the focus outline could be higher contrast, or more noticeable.

**Comment by rosejbon** [comment]
Here is Alex's check list - https://trello.com/c/WdGnhoof/278-accessibility-review-of-the-digital-maturity-matrix

**Comment by rosejbon** [comment]
I've updated the ca github with issues around accessibility for us to start working on. I've also added the card to make sure we are WCAG 2.1 AA compliant. Closing accessibility sweep as a result
